src/masoniteorm/migrations/Migration.py

```diff
@@ -118,8 +118,10 @@ def get_ran_migrations(self):
 
         database_migrations = self.migration_model.all()
         for migration in all_migrations:
-            if migration in database_migrations.pluck("migration"):
-                ran.append(migration)
+            if migration := database_migrations.where("migration", migration).first():
```

Comment:
oRM is currently supporting python 3.6+. this would break on 3.6 and 3.7 right?

Reply:
You're right, but this operator can be replaced. I will replace it to improve compatibility.